robot_controllers/setup.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+from catkin_pkg.python_setup import generate_distutils_setup
+
+d = generate_distutils_setup(
+    packages=["robot_controllers"],
+    package_dir={'': 'src'},
```

Comment:
you need to add an __init__.py file in src/robot_controllers/__init__.py or the install rules will fail